presto-native-execution/presto_cpp/main/PrestoTask.h

```diff
@@ -95,8 +95,14 @@ struct PrestoTask {
   /// has not been started, until the actual 'create task' message comes.
   bool taskStarted{false};
 
+  /// Time point (in ms) when the last message (any) came for this task.
```

Comment:
Can we add unit test in PrestoTaskTest.cpp to cover this heartbeat timestamp? Thanks!

Reply:
I don't believe we can easily organize a unit test for it.
This will require orchestrating TaskManager and re-implementing some sort of TaskResource to emulate HTTP messages.

Reply:
We can create a dummy PrestoTask and verify the newly added API works as expected? Thanks!

Reply:
That would a useless unit test, IMHO.
The functions are extremely simple - no space for error. I'm reluctant to add test for this not because I don't want it, but because we have so many already and this one would be near useless.
Like they are literally one-liners:
`lastCoordinatorHeartbeatMs = velox::getCurrentTimeMs();`
or
```
  if (lastCoordinatorHeartbeatMs == 0UL) {
    return 0UL;
  }
  return getCurrentTimeMs() - lastCoordinatorHeartbeatMs;
```
The bugs can sneak in only from the workflow in TaskManager, which would be really hard to re-create in a test.